MeNext/Layout/PlayingScreen.cs

```diff
@@ -26,17 +26,14 @@ public PlayingScreen(MainController mainController)
                 Command = new Command(() => mainController.RequestPrevious())
             });
 
+            var playCommand = new Command(() =>
+            {
+                mainController.musicService.Playing = !mainController.musicService.Playing;
```

Comment:
Per Daniel's comment, we shouldn't change the playing status through commands. All state updates happen through the server because the server is the truth.